netci.groovy

```diff
@@ -285,6 +285,41 @@ def osShortName = ['Windows 10': 'win10',
 }
 
 // **************************
+// Define perf testing.  These tests should be run daily, and will run on Jenkins not in Helix
+// **************************
+[true, false].each { isPR ->
```

Comment:
No, we do not.  Eventually, we want to run on PRs, but for now we will take this out.